CONTRIBUTING.md

```diff
@@ -23,6 +23,105 @@ First of all, thank you for contributing. It’s appreciated.
 
 ## Component structure
 
+### Create a new Component
+Here are some basic steps to get you started building your first component.
+
+1. Create your new component folder
+`src/components/MyNewComponent/`
+2. Create your component file(s)
+```
+    /MyNewComponent/
+      |-examples/
+        |-0.default.jsx <-- default example, used by `gulp docs-generate` to create component demo
+      |-MyNewComponent.jsx  <-- component logic goes here
+      |-MyNewComponent/MyNewComponent.less <-- component styling goes here
+      |-MyNewComponent/MyNewComponent.spec.js <-- component tests go here
+  ```
+3. Component Template
+```javascript
+import _ from 'lodash';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { lucidClassNames } from '../../util/style-helpers';
+import { createClass, findTypes, omitProps } from '../../util/component-types';
+
+const cx = lucidClassNames.bind('&-MyNewComponent');
+
+const { string, bool, node } = PropTypes;
+
+const MyNewComponent = createClass({
+	displayName: 'MyNewComponent',
+	propTypes: {
+		/**
+		 * Appended to the component-specific class names set on the root
+		 * element.
+		 */
+		className: string,
+	},
+
+	render() {
+		const {
+			className,
+		} = this.props;
+
+		return (<div className={className}> MyComponent </div>);
+	}
+});
+
+export default MyNewComponent;
+```
+4. Styling Template
+```css
+@import (reference) '../../styles/variables.less';
+@import (reference) '../../styles/mixins.less';
+
+.lucid-MyNewComponent {}
+```
+5. Tests Template
+```javascript
+import { common } from '../../util/generic-tests';
+import MyNewComponent from './MyNewComponent';
+
+describe('MyNewComponent', () => {
+	common(MyNewComponent);
+});
+```
+6. Default Example Template
+```javascript
+import React from 'react';
+import createClass from 'create-react-class';
+import { MyNewComponent } from '../../../index';
+
+/**
```

Comment:
I think this comment block is supposed to be in the actual component, not the examples